assets/node_modules/@enhavo/app/action/model/FilterAction.ts

```diff
@@ -17,8 +17,14 @@ export default class FilterAction extends DropdownAction
         for (let i in this.filterManager.filters) {
             let action = this.singleFilterActionFactory.createNew();
             action.filterKey = this.filterManager.filters[i].key;
-            action.label = this.filterManager.filters[i].label;
             action.setActive(this.filterManager.filters[i].active);
+
+            if (typeof this.filterManager.filters[i].label == "object" && this.filterManager.filters[i].label.hasOwnProperty('label')) {
```

Comment:
How about, we just pass the label as string and add further properties like `label_to` and `label_from` to the `BetweenFilter`? So we don't need this check at all